docs/DocumentingYourExtension.md

```diff
@@ -14,4 +14,11 @@ Properties in `docs/templates/includes/attributes.adoc` will be resolved and cop
 
 Consider adding documentation to the [Quarkiverse docs page](https://docs.quarkiverse.io/). 
 The Quarkiverse Hub uses [Antora](https://antora.org/) to aggregate each extension's documentation in the Quarkiverse docs website. 
-To register your extension's documentation, open a PR including it in the [antora-playbook.yml](https://github.com/quarkiverse/quarkiverse-docs/blob/main/antora-playbook.yml)
\ No newline at end of file
+To register your extension's documentation, open a PR including it in the [antora-playbook.yml](https://github.com/quarkiverse/quarkiverse-docs/blob/main/antora-playbook.yml)
+
+## Migration to Quarkus 3.14
+
+Before 3.14 the config documentation used to be generated in the root directory of your project.
+This is not the case anymore.
+Therefore, if you upgrade the Quarkus used in your Quarkiverse extention from a version older than 3.14 to any version newer than 3.13 you need to set up the `quarkus-config-doc-maven-plugin` in your project.
```

Comment:
The wording was a bit confusing with the reference to 3.13, I think given the context of the paragraph, we can just simplify things a bit. Also fixed a small typo.

```suggestion
Therefore, if you upgrade the Quarkus used in your Quarkiverse extension from a version older than 3.14, you need to set up the `quarkus-config-doc-maven-plugin` in your project.
```

Reply:
Well, I wanted to be accurate because if upgrade from 3.11 to 3.12 then it does not apply but you're right that it's ok in the context of the whole paragraph.